ai_eval/templates/coach_layout.html

```diff
@@ -0,0 +1,93 @@
+{% load i18n %}
+<div class="coach-block">
+    {% if intro_text %}
+        <div class="coach-intro">
+            {{ intro_text | safe }}
+        </div>
+    {% endif %}
+    {% if question_text %}
+        <div class="coach-question">
+            <div class="question-text">
+                {{ question_text | safe }}
+            </div>
+        </div>
+    {% endif %}
+
+    <div class="coach-layout">
+        <section class="coach-pane coach-pane--workspace" aria-label="{% blocktrans %}Learner workspace{% endblocktrans %}">
+            <header class="coach-pane__header">
+                <h2 class="coach-pane__title coach-pane__title--workspace">{{ self.workspace_title }}</h2>
+            </header>
+            <div class="coach-history" data-character-index="0" role="log" aria-live="polite" aria-relevant="additions text" aria-atomic="false">
+                <div class="coach-messages" data-pane="workspace"></div>
+                <div class="coach-spinner" aria-hidden="true">
+                    <span></span><span></span><span></span>
+                </div>
+            </div>
+            <div class="sr-only coach-status coach-status--workspace" role="status" aria-live="polite"></div>
+            <div class="coach-input" data-character-index="0">
+                <label for="coach-workspace-input" class="sr-only">{% trans "Start typing your answer" %}</label>
+                <textarea id="coach-workspace-input"
+                          class="coach-input__textarea"
+                          placeholder="{% trans 'Start typing your answer' %}"
+                          maxlength="1000"
+                          rows="1"
+                          data-character-index="0"></textarea>
+                <button type="button"
+                        class="coach-button coach-button--icon coach-send-button"
+                        aria-label="{% trans 'Send message' %}"
+                        data-character-index="0">
+                    <i class="fa fa-paper-plane" aria-hidden="true"></i>
```

Comment:
I found this behaviour confusing - what's the difference between the send button and "Submit for evaluation". For me, the send button used attempts as though I had submitted. 

Also, the "Submit for evaluation" button is either disabled, or it results in an error when I click it..